crates/sdl-encoder/src/field.rs

```diff
@@ -0,0 +1,114 @@
+use crate::FieldType;
+use std::fmt::{self, Display};
+
+/// Field in a given SDL type.
+#[derive(Debug, PartialEq, Clone)]
+pub struct Field {
+    description: Option<String>,
+    //TODO(@lrlna): fields for input objects can also take arguments. This
```

Comment:
To clarify, it's the other way around: fields on object types (and interfaces) can take arguments, but fields on input objects do not. Another difference is that input object fields can have default values. For these reasons, it might make sense to follow `graphql-js` and differentiate between `Field` and `InputField`.

Reply:
ok! thanks! I clarified the comment.

Reply:
Oh, another reason to differentiate between `Field` and `InputField` is that the type of an input object has to be an input type (meaning it can't be an object type/interface/union). (In `graphql-js` this is represented as a `GraphQLInputType`, but there may be a better way to encode this.)

Reply:
For now I'll keep `InputObject` to have a regular `Field` for now. Once we have an AST we can visit when encoding schemas, we can bring in an `InputField` as that would be a lot easier. I made a note specifically for this in `InputObject` struct definition.